package.json

```diff
@@ -21,14 +21,13 @@
 	],
   "dependencies": {
     "bootstrap": "^3.3.6",
-    "closure-templates": "20140422.0.0",
-    "metal": "metal/metal.js#master",
-		"metal-component": "metal/metal-component#master",
-		"metal-dom": "metal/metal-dom#master",
-		"metal-soy": "metal/metal-soy#master"
+    "metal": "^1.0.0-rc.1",
+    "metal-component": "1.0.0-rc.2",
```

Comment:
Oh, and just one other thing, make sure to always add the `^` prefix to these version numbers, so that the dependencies can be automatically updated through a new `npm i`, otherwise this component will always need to update the package.json to get new versions.